sdk/eventgrid/arm-eventgrid/test/eventgrid_examples.ts

```diff
@@ -72,15 +74,6 @@ describe("Eventgrid test", () => {
     assert.equal(resArray.length, 1);
   });
 
-  it("topics delete test", async function () {
-    const res = await client.topics.beginDeleteAndWait(resourceGroupName, topicName, testPollingOptions);
```

Comment:
why removing this case?

Reply:
I didn't remove it , I moved it down the order